modules/javafx.graphics/src/main/java/javafx/application/Platform.java

```diff
@@ -470,10 +470,9 @@ public static Preferences getPreferences() {
      * always available.
      * <p>
      * The following preferences are potentially available on the specified platforms:
-     * <table id="preferences-table">
-     *     <caption>List of platform preferences</caption>
+     * <table id="preferences-table" class="striped">
```

Comment:
I recommend changing the ID to `id="preferences-table-windows"` and adding  `id="preferences-table-macos"` and  `id="preferences-table-linux"` to the other two.

Reply:
And if you do that, you will need to change the link on line 448 of this file.